ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/verkle/VerkleAccount.java

```diff
@@ -37,6 +37,7 @@
 
 public class VerkleAccount extends DiffBasedAccount {
   private Hash storageRoot; // TODO REMOVE AS USELESS
+  private int hashCode;
```

Comment:
should we remove storageRoot while we are here

Reply:
good shout. @matkt mentioned he will remove it in future PR so I didn't want to create conflicts unnecessarily